packages/utils/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@extension/utils",
+  "version": "0.2.2",
+  "description": "chrome extension utils",
+  "private": true,
+  "sideEffects": false,
+  "files": [
+    "dist/**"
+  ],
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "index.ts",
+  "scripts": {
+    "clean": "rimraf ./dist && rimraf ./build",
+    "ready": "tsc",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "pnpm lint --fix",
+    "prettier": "prettier . --write --ignore-path ../../.prettierignore",
+    "type-check": "tsc --noEmit"
```

Comment:
If we add functions in the future, we'll need to set esbuild or something... How about adding a function example?